spark/common/src/test/scala/org/apache/sedona/sql/rasteralgebraTest.scala

```diff
@@ -1074,7 +1074,8 @@ class rasteralgebraTest extends TestBaseScala with BeforeAndAfter with GivenWhen
           }
           assert(rast2.getRenderedImage.getSampleModel.getDataType == dataType)
           val noDataValue = RasterUtils.getNoDataValue(rast2.getSampleDimension(0))
-          assert(noDataValue.isNaN)
+          val originalNoDataValue = RasterUtils.getNoDataValue(rast.getSampleDimension(0))
```

Comment:
The newly added original no data value is not used.

Reply:
Removed, thanks!